docs/census-open-data-registry.yaml

```diff
@@ -0,0 +1,104 @@
+Name: CZ CELLxGENE Discover Census
+Description: The Census provides efficient computational tooling to access, query, and analyze all single-cell RNA data from CZ CELLxGENE Discover.
+Documentation: https://chanzuckerberg.github.io/cellxgene-census/
+Contact: cellxgene@chanzuckerberg.com
+ManagedBy: "[Chan Zuckerberg Initiative Foundation](http://www.chanzuckerberg.com/)"
+UpdateFrequency: New releases are published weekly. LTS releases are published every 6 months.
```

Comment:
Should we expand LTS to `Long-term stable ("LTS")` for clarity?

Reply:
LTS means "Long Term _Support_" :-)